test/standardizedToken.js

```diff
@@ -0,0 +1,304 @@
+'use strict';
+
+const chai = require('chai');
+const { ethers } = require('hardhat');
+const {
+    Contract,
+    utils: { splitSignature, keccak256, toUtf8Bytes },
+    constants: { MaxUint256, AddressZero },
+} = ethers;
+const { expect } = chai;
+const { getRandomBytes32, getChainId, expectRevert, getGasOptions } = require('../scripts/utils');
+const { deployContract } = require('../scripts/deploy');
+
+const StandardizedToken = require('../artifacts/contracts/token-implementations/StandardizedToken.sol/StandardizedToken.json');
+const StandardizedTokenProxy = require('../artifacts/contracts/proxies/StandardizedTokenProxy.sol/StandardizedTokenProxy.json');
+
+describe('StandardizedToken', () => {
+    let standardizedToken, standardizedTokenDeployer;
+
+    const name = 'tokenName';
+    const symbol = 'tokenSymbol';
+    const decimals = 18;
+    const mintAmount = 123;
+
+    let token;
+    let tokenProxy;
+
+    let owner, user;
+
+    before(async () => {
+        const wallets = await ethers.getSigners();
+        owner = wallets[0];
+        user = wallets[1];
+    });
+
+    beforeEach(async () => {
```

Comment:
do we always need to deploy before each test? can you optimize it a bit to make it run faster?

Reply:
changed to `before` block